tools/src/main/resources/workflow.sh

```diff
@@ -5,9 +5,10 @@ FOLDER=${2:-${WORKFLOW_TOOLS_HOME}/conf}
 
 TOOLS_OPTS="-Dprovider.path=$FOLDER"
 export TOOLS_OPTS
+export JAVA_OPTS="-Xms2048m -Xmx2048m"
 
 "${WORKFLOW_TOOLS_HOME}/bin/export-workflow" download $WORKFLOW_NAME
 
 cd $FOLDER/$WORKFLOW_NAME
-"${INGESTION_HOME}/bin/flume-ng" agent --conf . --conf-file $WORKFLOW_NAME --name $WORKFLOW_NAME -f ./$WORKFLOW_NAME/$WORKFLOW_NAME -Dflume.root.logger=DEBUG,console -Dlog4j.configuration=file://${INGESTION_HOME}/conf/log4j.properties
+"${INGESTION_HOME}/bin/flume-ng" agent --conf . --conf-file $WORKFLOW_NAME --name $WORKFLOW_NAME -f ./$WORKFLOW_NAME/$WORKFLOW_NAME -Dflume.root.logger=DEBUG,console -Dlog4j.configuration=file://${INGESTION_HOME}/conf/log4j.properties $JAVA_OPTS
```

Comment:
would be nice to add JAVA_OPTS at the end of this line instead of -Xmx and -Xms params